src/components/Checkbox.module.css

```diff
@@ -23,8 +23,6 @@
   pointer-events: none;
   visibility: hidden;
   position: absolute;
```

Comment:
I think since you are using flex-box to position the checkmark, we can safely remove the position relative and absolute from the CheckboxContainer and the checkbox.

Reply:
- position absolute cannot be removed from the checkmark, because then it will be rendered next to the input, instead on top of it. The flexbox centers the elements, but centers them together, so one becomes left in the center other right in the center. By giving the checkmark a position absolute it is removed from the flow, both elements are now centered in isolation from each other.
- The position relative can be removed from the container, although I rather not, because I do not understand why this works. The absolute specs say the element will be positioned absolute to its first relative (or absolute) parent. Maybe it works because relative is also the default. I am rather explicit here, I think that is more clear.